Adding lxnet when lsocket and lnsl are included makes no sense.

--- mkspecs/solaris-g++/qmake.conf.orig	Wed Oct 30 19:12:45 2013
+++ mkspecs/solaris-g++/qmake.conf	Wed Oct 30 19:20:07 2013
@@ -28,7 +28,7 @@
 QMAKE_CFLAGS_SHLIB	= -fPIC
 QMAKE_CFLAGS_STATIC_LIB	= $$QMAKE_CFLAGS_SHLIB
 QMAKE_CFLAGS_YACC	= -Wno-unused -Wno-parentheses
-QMAKE_CFLAGS_THREAD	= -D_REENTRANT
+QMAKE_CFLAGS_THREAD	= -pthreads -D_REENTRANT
 
 QMAKE_CXX		= g++
 QMAKE_CXXFLAGS		= $$QMAKE_CFLAGS
@@ -42,10 +42,10 @@
 QMAKE_CXXFLAGS_YACC	= $$QMAKE_CFLAGS_YACC
 QMAKE_CXXFLAGS_THREAD	= $$QMAKE_CFLAGS_THREAD
 
-QMAKE_INCDIR		= /usr/sfw/include
-QMAKE_LIBDIR		= /usr/sfw/lib
-QMAKE_INCDIR_X11	= /usr/openwin/include
-QMAKE_LIBDIR_X11	= /usr/openwin/lib
+QMAKE_INCDIR		= /usr/g++/include
+QMAKE_LIBDIR		= /usr/g++/lib
+QMAKE_INCDIR_X11	= /usr/X11R6/include
+QMAKE_LIBDIR_X11	= /usr/X11R6/lib
 QMAKE_INCDIR_QT		= $$[QT_INSTALL_HEADERS]
 QMAKE_LIBDIR_QT		= $$[QT_INSTALL_LIBS]
 QMAKE_INCDIR_OPENGL	= /usr/openwin/include
@@ -63,7 +63,7 @@
 QMAKE_LFLAGS_SONAME	= -h$$LITERAL_WHITESPACE
 QMAKE_LFLAGS_THREAD	=
 QMAKE_LFLAGS_NOUNDEF	= -z defs
-QMAKE_LFLAGS_RPATH	= -Wl,-R,
+QMAKE_LFLAGS_RPATH	= -R
 
 QMAKE_LIBS		=
 QMAKE_LIBS_DYNLOAD	= -ldl
@@ -73,7 +73,7 @@
 QMAKE_LIBS_OPENGL	= -lGL
 QMAKE_LIBS_OPENGL_QT	= -lGL
 QMAKE_LIBS_THREAD	= -lpthread -lrt
-QMAKE_LIBS_NETWORK      = -lresolv -lsocket -lxnet -lnsl
+QMAKE_LIBS_NETWORK      = -lresolv -lsocket -lnsl
 
 QMAKE_MOC		= $$[QT_INSTALL_BINS]/moc
 QMAKE_UIC		= $$[QT_INSTALL_BINS]/uic
